parlai/agents/local_human/local_human.py

```diff
@@ -24,4 +25,11 @@ def act(self):
         reply = {}
         reply['id'] = self.getID()
         reply['text'] = input("Enter Your Reply: ")
+        if reply_text == '[DONE]':
```

Comment:
looks like something wrong here?

also if you use [DONE] to denote the episode is done,  i guess the text is empty?
we could also allow some text and a [DONE] at the end so we could send a final answer and "leave"